SharpGrad/Value.cs

```diff
@@ -43,6 +44,8 @@ public Value(double value)
 
             result._backward = () =>
             {
+                // y = x * b
+                // dy/dx = b
```

Comment:
<!-- Comment from Crocodile -->
Comment from [Crocodile 🐊](https://app.crocodile.dev/reviews/rwPWqdvJHMJe) on unchanged code outside of the pull request:

https://github.com/jlao/SharpGrad/blob/a6b2fd158657e26e99b29ea03d67040387729c43/SharpGrad/Value.cs#L58

Comment on existing code from Croc